portal-api/package.json

```diff
@@ -48,16 +48,16 @@
     "graphql": "^15.8.0",
     "graphql-middleware": "^6.1.33",
     "joi": "^17.6.0",
-    "jsonwebtoken": "8.5.1",
+    "jsonwebtoken": "^9.0.0",
```

Comment:
there are some breaking changes in v9 ->
https://github.com/auth0/node-jsonwebtoken/wiki/Migration-Notes:-v8-to-v9

Reply:
All tests have passed, are you happy for me to merge @IgnatG ?

Reply:
yup - happy to merge